libcxx/include/__hash_table

```diff
@@ -1771,9 +1771,9 @@ template <class _Tp, class _Hash, class _Equal, class _Alloc>
 template <class _Key>
 typename __hash_table<_Tp, _Hash, _Equal, _Alloc>::iterator
```

Comment:
Not attached to this file: Please add the before/after benchmarks in the commit message.

Reply:
Attached benchmarks data (with/without the opt) in the new commit.
Should I update the first comment of the conversation?